index.css

```diff
@@ -0,0 +1,143 @@
+*{
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.Main-Page {
+  max-width: 500px;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height:max-width;
+  width: 375px;
+  left: 0;
+  top: 92px;
+  border-radius: 0;
+  border: 2px solid black;
+  padding: 120px, 24px, 120px, 24px;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background: #3c3a39;
+  width: 375px;
+  height: 92px;
+}
+
+.section {
+  height: 590px;
+  width: 375px;
+  background: black;
+}
+
+.section img {
+  width: 100%;
+  height: 100%;
+}
+
+.logo {
+  color: white;
+  margin-top: 40px;
+  margin-left: 20px;
+  font-family: 'Poppins';
+  font-weight: 1000;
+text-decoration: none;
+  font-size: 20px;
+  text-align: Center;
+  Fill: Solid
+  #FFFFFF
+  
+}
+
+i {
+  font-size: 20px;
+  margin-right: 40px;
+  margin-top: 45px;
+  color: white;
+}
+
+h2 {
+  width: 327px;
+  height: 27px;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  color: #fff5e1;
+  display: flex;
+  align-items: center;
+  margin-left: 25px;
+}
+
+h1 {
+  margin-top: 100px;
+  margin-left: 20px;
+  width: 327px;
+  height: 124px;
+  font-family: 'Crete Round';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  color: #ff6b00;
+  letter-spacing: 0.37px;
+}
+
+.content {
+  width: 327px;
+  height: 123px;
+  font-family:'Poppins'; 
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  margin-top: 10px;
+  margin-left: 25px;
+}
+
+.imgcontainer {
+  display:flex;
+  flex-direction:row;
+  margin-left: 25px;
+  width: max-content;
+
+}
+
+img {
+  padding:5px;
+  margin-top: 0px;
+}
+
+#Medium{
+  padding-top: 15px;
+}
+
+/* .my-works {
+  display: grid;
+  background-color: white;
+  }
+  
+  .my-works-header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0px;
+  gap: 24px;
+  
+  width: 327px;
+  height: 107px;
+  
+  
+  /* Inside auto layout */
+  
+  /* flex: none;
+  order: 0;
+  flex-grow: 0;
+  }  */
```

Comment:
- [ ] Please remove these commented code from your project as requested by the previous reviewer.